src/config/index.js

```diff
@@ -21,6 +21,7 @@ const configuration = {
   ACCESS_TOKEN_COOKIE_NAME: process.env.ACCESS_TOKEN_COOKIE_NAME,
   USER_INFO_COOKIE_NAME: process.env.USER_INFO_COOKIE_NAME,
   NODE_ENV: process.env.NODE_ENV,
+  CUSTOMER_SUPPORT_EMAIL: 'customersuccess@edx.org',
```

Comment:
Do we need to define it like other `env variables` or is it good like this?

Reply:
I think @moconnell1453 can better answer that.
My understanding is that we are hardcoding the email address, so no need to expect it coming from the environment variables.

Reply:
I think this is acceptable, since the customer success email address is unlikely to change over time. It's hardcoded in a number of other places throughout this same repo.